behat.yml

```diff
@@ -2,11 +2,9 @@ default:
   suites:
     default:
       contexts:
-        - 'FeatureContext': { doctrine: '@doctrine' }
-        - 'HydraContext'
-        - 'Behat\MinkExtension\Context\MinkContext'
-        - 'Sanpi\Behatch\Context\RestContext'
-        - 'Sanpi\Behatch\Context\JsonContext'
+        - Behat\MinkExtension\Context\MinkContext
```

Comment:
Even with some feature, there's no reason at all to use Mink and another HTTP-related context.